osf/models/action.py

```diff
@@ -69,8 +70,16 @@ class RegistrationAction(BaseAction):
     to_state = models.CharField(
         max_length=31, choices=RegistrationModerationStates.char_field_choices())
 
+
 class SchemaResponseAction(BaseAction):
     target = models.ForeignKey('SchemaResponse', related_name='actions', on_delete=models.CASCADE)
     trigger = models.CharField(max_length=31, choices=SchemaResponseTriggers.char_field_choices())
     from_state = models.CharField(max_length=31, choices=ApprovalStates.char_field_choices())
     to_state = models.CharField(max_length=31, choices=ApprovalStates.char_field_choices())
+
+
+class CollectionSubmissionAction(BaseAction):
```

Comment:
Let's not worry about subclassing BaseAction since we need to redefine a lot of stuff anyway? Then we can go with the `deleted` standard instead of `is_deleted`? At some point, we can maybe fix the existing ones, since I suspect none of them have `is_deleted == True`